hummingbot/client/command/gateway_command.py

```diff
@@ -205,7 +205,8 @@ async def generate_certs(self):
     async def _api_request(self,
                            method: str,
                            path_url: str,
-                           params: Dict[str, Any] = {}) -> Dict[str, Any]:
+                           params: Dict[str, Any] = {},
+                           fail_silently = False) -> Optional[Dict[str, Any]]:
```

Comment:
code readability - add a ": bool" type hint after `fail_silently` to make it clear it's a boolean.